settings.ini

```diff
@@ -3,17 +3,17 @@ autoStart=1
 initialRes=100
 
 [Resolution change]
-minRes=60
+minRes=35
 maxRes=500
 dataPullDelayMs=250
 resChangeDelayMs=1400
-minGpuTimeThreshold=0.75
+minGpuTimeThreshold=1.3
 resIncreaseMin=3
 resDecreaseMin=9
 resIncreaseScale=60
 resDecreaseScale=90
-resIncreaseThreshold=75
+resIncreaseThreshold=70
```

Comment:
Why lower this back? I think increasing both thresholds by 3% seemed actually like a good idea.

Reply:
changed in the latest commit